extensions/google-chrome/src/hooks/useTabSearch.tsx

```diff
@@ -5,17 +5,21 @@ import { getPreferenceValues } from "@raycast/api";
 import { NOT_INSTALLED_MESSAGE } from "../constants";
 import { NotInstalledError, UnknownError } from "../components";
 import { usePromise } from "@raycast/utils";
+import { parseSearchQuery, matchesQuery } from "../util/search-parser";
 
 /**
  * @name useTabSearch
- * @description Filters chrome tabs where the url and title match all tab-or-space-separated words in search query (case insensitive).  Examples given title "foo bar" with url "example.com":
+ * @description Filters chrome tabs where the url and title match all tab-or-space-separated words in search query (case insensitive).
+ * Supports exclude terms with "/" prefix to filter out results containing those terms.
  * @example Given title "foo bar" with url "example.com":
  * search "foo bar" succeeds
  * search "bar foo" succeeds
- * search "foo example" succeds
- * search "example foo" succeds
+ * search "foo example" succeeds
+ * search "example foo" succeeds
  * search "foo" succeeds
  * search "example" succeeds
+ * search "foo /bar" succeeds (contains foo but not bar)
+ * search "/example" fails (excludes example.com)
  * search "asdf" fails
  */
```

Comment:
This comment references cases now handled by query-parser. Please move the comment there too.